.NET/Microsoft.Recognizers.Definitions.Common/Japanese/DateTimeDefinitions.cs

```diff
@@ -322,6 +332,16 @@ public static class DateTimeDefinitions
             { @"3", 3 },
             { @"4", 4 },
             { @"5", 5 },
+            { @"０", 0 },

```

Comment:
This should not be necessary. Same in other maps.

Reply:
done

Reply:
removed